models/work-profiles.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
```

Comment:
You can import it doing the following:
```suggestion
const { Schema } = mongoose;
```

Reply:
I didn't know that it can be imported like this. Implemented ✅